zanata-war/src/main/java/org/zanata/security/SpNegoIdentity.java

```diff
@@ -54,9 +54,7 @@ public void authenticate() {
         ZanataIdentity identity =
                 ServiceLocator.instance().getInstance(ZanataIdentity.class);
         if (identity.isLoggedIn()) {
-            if (Events.exists()) {
-                Events.instance().raiseEvent(Identity.EVENT_ALREADY_LOGGED_IN);
-            }
+            getAlreadyLoggedInEvent().fire(new AlreadyLoggedInEvent());
```

Comment:
How is this working in tests, if it doesn't check whether Event scope is active?  Did we not need the if check at all?